a11y/a11y.test.js

```diff
@@ -0,0 +1,67 @@
+import fs from 'fs'
+import path from 'path'
+
+import puppeteer from 'puppeteer'
+import execa from 'execa'
+
+import { mockServer } from './mock-server'
+
+const OTP_RR_CONFIG_FILE_PATH = './config.yml'
+const OTP_RR_CONFIG_BACKUP_PATH = './config.non-test.yml'
+const OTP_RR_TEST_CONFIG_PATH = './a11y/test-config.yml'
+
+beforeEach(() => {
+  // backup current config file
+  if (fs.existsSync(OTP_RR_CONFIG_FILE_PATH)) {
+    fs.renameSync(
+      OTP_RR_CONFIG_FILE_PATH,
+      OTP_RR_CONFIG_BACKUP_PATH
+    )
+    console.log('Backed up current OTP-RR config file')
+  }
+  // copy over test config file
+  fs.copyFileSync(
+    OTP_RR_TEST_CONFIG_PATH,
+    OTP_RR_CONFIG_FILE_PATH
+  )
```

Comment:
This bit of code should be the place where tokens and live endpoints should be inserted from environment variables.

Reply:
The mock server removes the need for any of these keys. Sorry again for leaking them. Will keep my guard up more next time.